docker/build.Dockerfile

```diff
@@ -87,12 +87,37 @@ RUN  CXX_COMPILER="g++" \
     && cmake --build . --target check \
     && cmake --build . --target install
 
-# Enable perfomance layers and test with `vkcube`.
+# Enable the perfomance layers and test them with `vkcube`.
 RUN export LD_LIBRARY_PATH=/performance-layers/build:$LD_LIBRARY_PATH \
     && export VK_INSTANCE_LAYERS=VK_LAYER_STADIA_pipeline_compile_time \
     && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_runtime \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_cache_sideload \
-    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_memory_usage \
     && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_frame_time \
+    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_memory_usage \
+    && export VK_INSTANCE_LAYERS=$VK_INSTANCE_LAYERS:VK_LAYER_STADIA_pipeline_cache_sideload \
     && export VK_LAYER_PATH=/performance-layers/layer \
-    && xvfb-run vkcube --c 100 && echo Done!
+    && export VK_COMPILE_TIME_LOG=/performance-layers/build/compile_time.csv \ 
```

Comment:
In case the build succeeds and someone wants to extract the artefacts, I think it would be better not to write anything to the build directory. We can instead create a new temporary output directory with `mktmep -d`.

Reply:
That's a great idea.